packages/components/src/components/form/Select/Select.tsx

```diff
@@ -179,13 +179,24 @@ const Wrapper = styled.div<WrapperProps>`
         display: flex;
         align-items: center;
         justify-content: ${({ $isClean }) => ($isClean ? 'flex-end' : 'flex-start')};
-        width: 100%;
+        width: ${({ $isSearchable }) => !$isSearchable && '100%'};
```

Comment:
this will result into invalid css in case that isSearchable is false

Reply:
Noted, thanks. Had thought Styled would handle this and not render the property at all. I will fix it:)